Packages/Settings/Sources/SettingsUI/Internal/GitHubSettings/GitHubSettingsView.swift

```diff
@@ -77,6 +99,19 @@ private extension GitHubRunnerScope {
             L10n.Settings.RunnerScope.organization
         case .repo:
             L10n.Settings.RunnerScope.repository
+        case .enterpriseServer:
+            "Enterprise"
+        }
+    }
+}
+
+private extension GitHubServiceVersion.Kind {
+    var title: String {
+        switch self {
+        case .dotCom:
+            return "github.com"
+        case .enterprise:
+            return "github self hosted"
```

Comment:
I'm unfamiliar with enterprise GitHub setups and self-hosting so please correct me if I'm wrong but the `enterprise` enum case seems to refer to GitHub Enterprise Server. Would it make sense to rename the case and the title to be more explicit about this and avoid confusion with the enterprise runner scope?